modules/uncertainty/DESCRIPTION

```diff
@@ -43,7 +43,9 @@ Imports:
     plyr (>= 1.8.4),
     purrr,
     randtoolbox,
-    rlang
+    rlang,
+    PEcAn.data.atmosphere,
+    PEcAnAssimSequential
```

Comment:
Nitpicking: Can you move these lines up to stay in alphabetical order, please?

Reply:
so I am removing the dependency on these packages so they are no longer in the DESCRIPTION file.